package.json

```diff
@@ -35,6 +35,7 @@
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "^4.0.0",
     "ember-source": "~2.14.1",
+    "ember-truth-helpers": "^1.3.0",
```

Comment:
I don't see any of this imported. Do we need `ember-truth-helpers`?

Reply:
Yes, I prefer to use ember-truth-helpers. It makes conditional templating much easier.